README.md

```diff
@@ -8,6 +8,7 @@
 4. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
     * [Facts](#facts)
     * [Types/Providers](#types/providers)
```

Comment:
Types and providers rather than Types/Providers

Usually we list types and providers separately, though I'm fine with listing them together if that makes the most sense. Are providers to be added later? If there's a provider for each type, we could include that information in the type descriptions instead. Feel free to ping me to chat about how to handle this bit! :-)